readthedocs/settings/base.py

```diff
@@ -793,22 +816,26 @@ def DOCKER_LIMITS(self):
         'version': 1,
         'disable_existing_loggers': True,
         'formatters': {
-            'default': {
-                'format': LOG_FORMAT,
-                'datefmt': '%d/%b/%Y %H:%M:%S',
+            "plain_console": {
+                "()": structlog.stdlib.ProcessorFormatter,
```

Comment:
This `()` key feels weird to me.. but I assume it works?

Reply:
Seems to be a special magic ( 🪄 ) key from Python for user-defined objects: https://docs.python.org/3/library/logging.config.html#logging-config-dict-userdef